examples/sdp_examples/Converting a non-dcp compliant problem to Semi-Definite Programming Problem.ipynb

```diff
@@ -0,0 +1,111 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "#  Non-DCP Compliant Problem to Semi-Definite Programming Problem \n",
+    "\n",
+    "Consider the problem\n",
+    "\n",
+    "minimize $\\frac{(c^T x)^2}{d^Tx}$\n",
+    "\n",
+    "subject to Ax +b$\\geq$ 0\n",
+    "\n",
+    "Also $d^Tx$>0 whenever Ax+b$\\geq$ 0\n",
+    "\n",
+    "**It is very easy to obseve that the objective function is neither concave or convex or affine and hence it is not dcp compliant problem.**\n",
+    "\n",
+    "** exp = square(c'*x)/d'*x **    \n",
```

Comment:
The formatting of this whole section is weird. Did you run the code in the .ipynb or copy-paste it from a terminal? The former method is preferred.